client-misk/src/main/kotlin/app/cash/backfila/client/misk/BackfilaModule.kt

```diff
@@ -24,13 +26,16 @@ import misk.inject.KAbstractModule
 class BackfilaModule(
   private val config: BackfilaClientConfig,
   @Deprecated(message = "Multibind backfills instead")
-  private val backfills: List<KClass<out Backfill<*, *, *>>>? = null
+  private val backfills: List<KClass<out Backfill<*, *, *>>>? = null,
+  private val loggingSetupProvider: KClass<out BackfilaClientLoggingSetupProvider> =
```

Comment:
this doesn't seem very "guicey"

Reply:
Yeah. Maybe @swankjesse has a better idea...

Reply:
seems fine honestly

Reply:
Ok I'm going to run with this then